src/Common/ServerUUIDFile.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <functional>
+#include <string>
+#include <boost/noncopyable.hpp>
+
+
+namespace DB
+{
+class WriteBuffer;
+
+
+/** Provides that no more than one server works with one data directory.
+  */
+class ServerUUIDFile : private boost::noncopyable
```

Comment:
The class should allow us to obtain UUID otherwise it has no use.

Reply:
You're right, the class wasn't doing anything special. Removed it and simplified the logic to be done directly inside the `Server.cpp` file.
Can probably add a class later if there's a need to read the UUID - right now I don't read it anywhere..